website/source/partials/builder/amazon/common/_BlockDevice-not-required.html.md

```diff
@@ -0,0 +1,43 @@
+<!-- Code generated from the comments of the BlockDevice struct in builder/amazon/common/block_device.go; DO NOT EDIT MANUALLY -->
+
+-   `delete_on_termination` (bool) - Indicates whether the EBS volume is deleted on instance termination.
+    Default false. NOTE: If this value is not explicitly set to true and
+    volumes are not cleaned up by an alternative method, additional volumes
+    will accumulate after every build.
+    
+-   `device_name` (string) - The device name exposed to the instance (for example, /dev/sdh or xvdh).
+    Required for every device in the block device mapping.
+    
+-   `encrypted` (*bool) - Indicates whether or not to encrypt the volume. By default, Packer will
```

Comment:
the star `*` is doing this; hopefully in this case this will be fixed by your Trilean PR, but there are also some `*string` fields.

May be it's a good excuse to start doing a NilableString type 🤔.

Reply:
There I fixed the pointer parameters generation so that stars are escaped, this should fix the italics issue for github